openpype/hosts/nuke/startup/custom_write_node.py

```diff
@@ -0,0 +1,81 @@
+import os
+import nuke
+from openpype.hosts.nuke.api.lib import set_node_knobs_from_settings
+
+
+frame_padding = 5
+temp_rendering_path_template = (
+    "{work}/renders/nuke/{subset}/{subset}.{frame}.{ext}")
+
+knobs_setting = {
+    "knobs": [
+        {
+            "type": "text",
+            "name": "file_type",
+            "value": "exr"
+        },
+        {
+            "type": "text",
+            "name": "datatype",
+            "value": "16 bit half"
+        },
+        {
+            "type": "text",
+            "name": "compression",
+            "value": "Zip (1 scanline)"
+        },
+        {
+            "type": "bool",
+            "name": "autocrop",
+            "value": True
+        },
+        {
+            "type": "color_gui",
+            "name": "tile_color",
+            "value": [
+                186,
+                35,
+                35,
+                255
+            ]
+        },
+        {
+            "type": "text",
+            "name": "channels",
+            "value": "rgb"
+        },
+        {
+            "type": "text",
```

Comment:
Lets remove the colorspace knob. It would be better if the colorspace value will be taken from the workfile defaults. I had just tested it on a project set to ACES and this didn't work that well.